db/migrate/20150825092700_create_speech.rb

```diff
@@ -0,0 +1,16 @@
+class CreateSpeech < ActiveRecord::Migration
+  def change
+    create_table :speeches do |t|
+      t.string :name
+      t.text :description
+      t.belongs_to :user, index: true
+      t.string :slides
+      t.integer :expected_duration
+      t.timestamp :speech_time, index: true
+      t.integer :status
+      t.integer :category, index: true
```

Comment:
How do you think to use string to represent status?

I feel it can make database more readable. And storing string or number doesn't matter performance so much.